### DISCUSSION TITLE
Help needed on how to use showNotificationControls prop

### DISCUSSION THREAD

**Discussion by anasvemmully** [TheWidlarzGroup/react-native-video#3857] [Q&A]
### What are you trying to do?

Help needed on how to use the `showNotificationControls` prop (https://thewidlarzgroup.github.io/react-native-video/component/props#shownotificationcontorols).

I am new to react native. I want to use `showNotificationControls`, but I can't figure out how to use it correctly. Can anyone help?

### What have you tried?

I set the value to `true` for `showNotificationControls`, but I didn't see any change. I looked through the documentation and searched for any help on its usage, but I didn't find anything.

**Comment by freeboub** [comment]
There is a typo in prop name ... here is a fix: https://github.com/TheWidlarzGroup/react-native-video/pull/3859/files
Di you try to test the sample app, it uses this feature !
but please fix the typo issue in your app ...

**Reply by anasvemmully** [reply]
Sorry for the typo, i tried but I didn't find on my physical device (Realme 3i android 10)

![image](https://github.com/TheWidlarzGroup/react-native-video/assets/69687301/81cdd470-aac2-4040-8e2b-3e56709d7bb6)

![image](https://github.com/TheWidlarzGroup/react-native-video/assets/69687301/c953ba93-77e9-4609-8c28-5bf55107e75f)


I didn't see any controls

**Reply by KrzysztofMoch** [reply]
So you can see notification from "VideoPlayer" it should be there but for some reason isn't ... 🤔

**Reply by anasvemmully** [reply]
yah for some reason i dont know the notification didn't even show up, at least I can see the notification in the example app, but not the controls!

**Reply by freeboub** [reply]
It would be interesting to test with another device...

**Comment by KrzysztofMoch** [comment]
Hey @anasvemmully, I've confirmed that this is an lib issue and this should be fixed by https://github.com/TheWidlarzGroup/react-native-video/pull/3886

**Reply by anasvemmully** [reply]
Great
